react-native-scripts/src/scripts/init.js

```diff
@@ -167,38 +167,85 @@ We recommend using npm >= 5.7.0 or yarn.
   } else {
     cdpath = appPath;
   }
+  // 'start'  -> npm run start / yarn start
+  const transformCommand = (packager, cmd) => {
+    if (packager === 'yarn') {
+      return packager + ' ' + cmd;
+    }
+    return packager + ' run ' + cmd;
+  };
 
-  log(
-    `
-Success! Created ${appName} at ${appPath}
-Inside that directory, you can run several commands:
-
-  ${chalk.cyan(npmOrYarn + ' start')}
-    Starts the development server so you can open your app in the Expo
-    app on your phone.
-
-  ${chalk.cyan(npmOrYarn + ' run ios')}
-    (Mac only, requires Xcode)
-    Starts the development server and loads your app in an iOS simulator.
-
-  ${chalk.cyan(npmOrYarn + ' run android')}
-    (Requires Android build tools)
+  const cyanOutputCmd = string => chalk.cyan(string);
+
+  // Output configuration for items to be printed to terminal
+  // 'desc' will resolve to a one-liner
+  // 'desc' and 'title' will make a pair of a command
+  // 'onShow' outputs the pair or one-liner based on a condition
+  const stdoutOutputConfig = {
+    onSuccessmsg: {
+      desc: `\nSuccess! Created ${appName} at ${appPath}` +
+        `\n` +
+        `Inside that directory, you can run several commands:\n`,
+    },
+    start: {
+      title: cyanOutputCmd(transformCommand(npmOrYarn, 'start')),
+      desc: `Starts the development server so you can open your app in the Expo
+    app on your phone.`,
+    },
+    ios: {
+      title: cyanOutputCmd(transformCommand(npmOrYarn, 'ios')),
+      desc: `(Mac only, requires Xcode) 
+    Starts the development server and loads your app in an iOS simulator.`,
+    },
+    android: {
+      title: cyanOutputCmd(transformCommand(npmOrYarn, 'android')),
```

Comment:
Can change this if you want, but it allows you to change colors or what you want pretty easily